packages/page-accounts/src/modals/ProxyOverview.tsx

```diff
@@ -147,22 +147,22 @@ function NewProxy ({ index, onChangeAccount, onChangeType, onRemove, proxiedAcco
   );
 }
 
-function getProxyTypeInstance (api: ApiPromise, index = 0): NodeRuntimeProxyType {
+function getProxyTypeInstance(api: ApiPromise, index = 0): NodeRuntimeProxyType {
   // This is not perfect, but should work for `{Kusama, Node, Polkadot}RuntimeProxyType`
   const proxyNames = api.registry.lookup.names.filter((name) => name.endsWith('RuntimeProxyType'));
 
   // fallback to previous version (may be Substrate default), when not found
   return api.createType<NodeRuntimeProxyType>(proxyNames.length ? proxyNames[0] : 'ProxyType', index);
 }
 
-function getProxyOptions (api: ApiPromise): { text: string; value: number; }[] {
+function getProxyOptions(api: ApiPromise): { text: string; value: number; }[] {
```

Comment:
```suggestion
function getProxyOptions (api: ApiPromise): { text: string; value: number; }[] {
```

Fix, CI fails linting due to this.